### ISSUE TITLE
Installation CLI - Database not exists! Please create database for your application before proceeding.

### ISSUE THREAD

**Issue by kidino** [akukoder/fortify-tabler-admin#3]
Running `php artisan fortify:tabler` after selecting some options I get the following error.

```
Database not exists! Please create database for your application before proceeding.
```

[![database-not-found.png](https://i.postimg.cc/R05tkQT5/database-not-found.png)](https://postimg.cc/yDjW9R5v)

I am certain that I have the .env updated with the correct DB details, and is working properly with other Controllers and Models.

### Laravel Version
9.8.1

### PHP Version
8.1.3

**Comment by syahzul** [comment]
Sorry, I'm unable to replicate this issue, but I've added option to skip running migration command.

Try composer update and re-run the command with ```--skip-migration``` option.

```php artisan fortify:tabler --skip-migration```

**Comment by kidino** [comment]
Nevermind... I know what it is. I already have Laravel Voyager installed, which normally can work well with Fortify and FortifyUI. But I think with your package, which uses *Arcanedev\LaravelSettings*, there is a conflict when trying to create the `settings` table at the `migrate` command.

Laravel Voyager already has a table with that name.

I think for now, maybe we can conclude that this is not compatible with Laravel Voyager.

**Comment by kidino** [comment]
I finally managed to install that package. Here's what I did.

After I encounter the error, I updated `config/settings.php:37` to add a prefix to the new `settings` table

```
                'table'      => 'fuitabler_settings',
```

Then I edited `2020_12_13_155612_update_users_table.php` which added the `avatar` column, which already existed. I removed that part where there `avatar` column was added.

Then I ran `php artisan migrate`

After that, I encountered error with email verification view. Need to find how to fix that. But since this is just my local dev, I just added some dates in the `email_verified_at` column. Now I can see the Tabler dashboard.
